src/messages/ui.en.json

```diff
@@ -219,9 +219,9 @@
 
   "languageDialog.title": "Language",
 
-  "popup.compatibility.bluetooth.header": "Bluetooth incompatible browser!",
-  "popup.compatibility.bluetooth.explain": "The browser you are currently using does not support Bluetooth.",
-  "popup.compatibility.bluetooth.advice": "Please update the browser or use another one from our supported browsers list below.",
+  "popup.compatibility.header": "This browser is not supported",
+  "popup.compatibility.explain": "Unfortunately, WebUSB and Web Bluetooth are not supported by this browser. This means you cannot connect connect to a micro:bit to record the data samples needed to train a machine learning model.",
```

Comment:
Looks good apart from the "connect connect".

Reply:
Oops. Fixed.